### ISSUE TITLE
Quake Rerelase assets not used

### ISSUE THREAD

**Issue by HarbingerDawn** [sultim-t/vkquake-rt#25]
Despite precisely following the instructions to install **only** the files from the Quake Rerelease folder, only the original assets are used. Is this intentional behavior? If so, what benefit is there to installing from the rerelease folder as opposed to the classic Quake files?

**Comment by Berzerk2k2** [comment]
Yep can confirm the rerelease models are not active. On top of that weapon models are also clipping through walls.

[Quake RTX weapon model glitch](https://imgur.com/a/oK35ZMe)

**Comment by Dragon-Mom** [comment]
I'm guessing it's because they'd have to edit the assets directly to make them compatible with the RTX reflections so instead it just doesn't use them, but the Quake Remastered models look so good it's hard for me to go back to the originals personally. Hoping this is added, or addressed if it's a bug.

**Comment by TR2NLG** [comment]
Confirming re-release models not working.

**Comment by TheEngiGuy** [comment]
Can confirm.

**Comment by Laithan401** [comment]
Confirmed the same issue here also. **_It seems to be reading from the original steam folder no matter what._** The workaround does not work either. When you click "no" the game will load and using RT however it is not running the rerelease version. You can tell right away that you don't get the new menu options and local LAN play doesn't work with someone running the rerelease.

**Comment by TheEngiGuy** [comment]
Does anyone have a fix for this yet?

**Comment by Laithan401** [comment]
I hope we can get a fix.. Quake is worth it (such a classic) and playing it using RT with the new version would be awesome.

**Comment by SonicBoomin** [comment]
> Does anyone have a fix for this yet?

Still pushing this topic along. I have same issue and wondering the same thing... Can someone remedy this?
